### PR TITLE
Create Zowe Explorer Mission

### PR DESCRIPTION
**Pull request by venkatzhub**
As the mission statement got lost in the formation of the squad process, I'm taking a stab at defining the mission/charter for consideration by the newly formed squad.

### CODE REVIEW

**Comment by solsu01** [comment]
Thanks @venkatzhub. We hope to have the squad lead settled by end of this week and they can be added as a reviewer to this PR. Will keep you updated.

File: Zowe Explorer Mission [hunk 1]
```diff
@@ -0,0 +1,8 @@
+As the mission statement got lost in the formation of the squad process, I'm taking a stab at defining the mission/charter for consideration by the newly formed squad.
```
Comment (solsu01) [inline Zowe Explorer Mission:1]: I would leave this out as it seems more like a comment on the reason for the change. That can be in the PR description.

Comment (venkatzhub) [reply Zowe Explorer Mission]: Done.

**Comment by phaumer** [comment]
@venkatzhub looks great. Thanks.

**Comment by solsu01** [comment]
@zFernand0 as the new squad for zowe explorer, feel free to merge this in if you and the squad are on board with it.

**Comment by venkatzhub** [comment]
As long as we are good with the content and it's reflected as the sub project mission, I'm good what ever format / form this gets incorporated in.

**Comment by venkatzhub** [comment]
The updated Readme now has two Zowe Explorer sections, is that the intent? I think it would make sense to consolidate everything under one "Zowe Explorer" section.

**Comment by solsu01** [comment]
@venkatzhub the first section is the sub-project and the second one is the squad description.

In Zowe, a single squad can have ownership of one or more sub-projects, so I've updated the document to reflect this. In the case of the Zowe Explorer squad, they have responsibility over the Zowe Explorer sub-project.

**Comment by venkatzhub** [comment]
Thanks @solsu01 .

**Pull request merged**